README.md

```diff
@@ -6,3 +6,20 @@ Nebulaworks Engineering use of [nixpkgs](https://github.com/NixOS/nixpkgs)
 
 * [Container Images](./imgs/README.md)
 * [Publishing](./PUBLISHING.md)
+* [iso Configs](./isos/README.md)
+
+## Resolving Nix Configurations
+While there are numerous ways of resolving Nix configs and derivations, one way to perform derivations in a non-linux environment is leveraging a Docker container. The general syntax for this approach is:
+```
+docker run --rm -it -v $(git rev-parse --show-toplevel):/build -w /build <dockerImage_withNix> /bin/sh -c "\
```

Comment:
I think here lets just leave it as `/bin/sh` for the command. That way it just drops you into the container with the nix environment loaded to do whatever you want.

Reply:
Good point, @sarcasticadmin, plus it also makes that part of the doc a lot cleaner.